### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kkarunakaran89**
1) Adding meeting timing and cadence
2) Adding Zoom link 
3) Adding link to notes and videos

Signed-off-by: kkarunakaran89 <109688962+kkarunakaran89@users.noreply.github.com>

### CODE REVIEW

**Review by SteveLasker — COMMENTED** [review]
Thanks @kkarunakaran89 
I was just doing something similar for the go-cose community page, where I added a line for the ics: https://github.com/veraison/go-cose/pull/127/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R26

The ics for the scitt-community call would be: https://zoom.us/meeting/tZIucuisqj4sGNYoptUH4YDWxty7rLcEz5eH/ics

**Comment by kkarunakaran89** [comment]
> Thanks @kkarunakaran89 I was just doing something similar for the go-cose community page, where I added a line for the ics: https://github.com/veraison/go-cose/pull/127/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R26
> 
> The ics for the scitt-community call would be: https://zoom.us/meeting/tZIucuisqj4sGNYoptUH4YDWxty7rLcEz5eH/ics

Hi Steve, great catch! I have updated the link

File: profile/README.md [hunk 1]
```diff
@@ -10,8 +10,9 @@ To work as a community to facilitate adoption of IETF Supply Chain Integrity, Tr
 + Join the SCITT community mailing list at https://groups.io/g/scitt-community
 
 ## SCITT Community Meetings:
-+ SCITT Community Meetings: We are working on scheduling our kickoff meeting. Cadence and time of meeting to be discussed during kickoff meeting
-- Link to meeting agenda, prior meeting notes and meeting videos will be made available
++ SCITT Community Meetings: Bi-Weekly on Wednesday 8AM PST/4PM UTC
++ SCITT Community Bi-Weekly Meeting (iCal link): [Click Here](https://zoom.us/meeting/tZIucuisqj4sGNYoptUH4YDWxty7rLcEz5eH/ics)
++ Meeting  Notes and Videos: [See link to meeting notes and previously recorded sessions](https://docs.google.com/document/d/1OBfJmZdxXfJ7MWHKmCaAn0Grn1LHxFOyFtPdH_U_dOQ/edit#)
```
Comment (yogeshbdeshpande) [inline profile/README.md:15]: Thanks for the PR
Minor point: Do we want to give edit permission to every one? 
The link points to edit# hence asking.

Rest all changes LGTM!

**Review by yogeshbdeshpande — APPROVED** [review]

File: profile/README.md (hunk 1 above)
Comment (SteveLasker) [reply profile/README.md:15]: It's the default mode, and doesn't change permissions, but it's a good point to cleanup. Doing now...

**Review by SteveLasker — APPROVED** [review]
LGTM with minor point to cleanup the default edit parameter in the hackmd link

**Review by OR13 — APPROVED** [review]

**Pull request merged**